spec/requests/api/v1/auth_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'API V1 Auth', type: :request do
+  let!(:users) { create_list(:user, 10) }
+  it 'should not allow access via http' do
```

Comment:
is this test about http or fetching data without token in header? I would like to see both

Reply:
It was a check that it should not allow access without authentication (token)